lib/galaxy/tools/imp_exp/export_history.py

```diff
@@ -90,16 +89,18 @@ def create_archive(history_attrs_file, datasets_attrs_file, jobs_attrs_file, out
 
         # Status.
         print('Created history archive.')
+        return 0
     except Exception as e:
         print('Error creating history archive: %s' % str(e), file=sys.stderr)
+        return 1
```

Comment:
What do you think about being explicit here and doing `sys.exit("Error creating history archive: %s" % str(e))` ?
This will print to stderr and exit with 1

Reply:
I prefer non-explicit exits, feels like an implementation detail that should be left to the interpreter. The way it is structured now makes it easier to use (and test) as a library function - I think the unit tests would probably break if I did this?